react_app/src/pages/Map/components/Cart.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Button, ListGroup, Modal } from 'react-bootstrap';
+
+const Cart = ({ authStatus, filteredBioSamples }) => {
+    const [show, setShow] = useState(false);
+
+    const handleClose = () => setShow(false);
+    const handleShow = () => setShow(true);
+
+    const backendUrl = process.env.REACT_APP_BACKEND_URL;
+    console.log('Filtered Biosamples in Cart:',filteredBioSamples);
+
+    return (
+        <div className="Cart Button" style={{ backgroundColor: '#f4f4f4', padding: '20px', boxShadow: '0 0 10px rgba(0, 0, 0, 0.1)', borderRadius: '8px' }}>
```

Comment:
It is better to follow the naming convention for class names:
- Use lowercase letters for all class names
- Use hyphens to separate words in class names